oak/server/rust/oak_runtime/src/node/grpc/server/mod.rs

```diff
@@ -51,6 +54,8 @@ pub struct GrpcServerNode {
     address: SocketAddr,
     /// Loaded files containing a server TLS key and certificates.
     tls_identity: Identity,
+    /// OpenID Connect Authentication client information.
```

Comment:
Clarify what is the meaning of this being None.

Reply:
Done.